swagger.yaml

```diff
@@ -4021,6 +4020,25 @@ definitions:
       type:
         $ref: '#/definitions/image_type'
 
+  host_static_network_config:
+    type: object
+    properties:
+      network_yaml:
+        type: string
+        description: yaml string that can be processed by nmstate
+      mac_interface_map:
+        $ref: '#/definitions/mac_interface_map'
+
+  mac_interface_map:
```

Comment:
pls include description for this property, since it has a specific assumption about the values provided in it and in the network_yaml